### PR TITLE
Add deprecated function query. This is needed for the release 3.2

### PR DESCRIPTION
**Pull request by tunacicek**
## Description
Add deprecated function query. This is needed for the release 3.2

### CODE REVIEW

**Comment by bs-sili** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.3.15-M1
+### Added
+
+## fixed
+- Add deprecated api `/query` for the release 3.2. The Api `query` is not a part of AAS 3.0. But for  release 3.2, this api will be added and marked as deprecated. This api will be removed in future releases.
```
Comment (agg3fe) [inline CHANGELOG.md:11]: nit: Shouldn't this be in 'Added' section?

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by bs-sili** [comment]
LGTM

**Pull request merged**